Automattic/wp-calypso
# Add an environment dropdown in the site dashboard

Instead of displaying a separate site entry for staging sites, we now display group sites with their corresponding staging site in a single entry. This is done by adding a dropdown with the site environment containing two options: Production and Staging.

When the user selects the staging option, they will be sent to the staging site detail page.

Related to https://github.com/Automattic/dotcom-forge/issues/7536
![Screenshot 2024-06-28 at 18 30 32](https://github.com/Automattic/wp-calypso/assets/995955/cf8e59dd-faf0-4d8d-81e1-31f14cac369d)
![Screenshot 2024-06-28 at 11 01 55](https://github.com/Automattic/wp-calypso/assets/995955/94c6c437-4a23-4db2-aceb-5347caec563e)



## Proposed Changes

* Add a new Environment dropdown in the Sites Dashboard
* Filter staging sites from the site list

## Why are these changes being made?


* Having separate entries for staging sites is confusing.

## Testing Instructions



* Use the live link
* Go to /sites and select an Atomic site
* Create a staging site
* Use the new Dropdown in the menu and navigate between staging and production site
* When you select an option, you'll be redirected to the corresponding environment
* On a staging site, go to staging tab and click on manage production site
* Notice that you're redirected to the production site
* Make sure that the design looks the same as the one provided in the issue Figma file (except for the badge in site list view)

## Pre-merge Checklist
